### DISCUSSION TITLE
myMPD v7.0.2  = Stable release

### DISCUSSION THREAD

**Discussion by vernonjvs** [jcorporation/myMPD#638] [General]
First, I want to thank you for myMPD. myMPD v7.0.2 works flawlessly on all the Linux and IOS browsers that I use. I don't have any Windows or Android machines so I can't speak for those.

However, v8+ broke some features that I use. So, if you are a new myMPD user, like I am, and you are having issues with the v8+ releases, you may want to try the v7.0.2 release which is what I use as my daily driver while the v8+ bugs get sorted.

A couple of remaining bugs in the v8.2 development and probably prior v8 releases which are not present in the v7.0.2 release. 

- myMPD development is not displayed properly on Firefox 78.15.0esr. Please see image below for details
![firefox-78 15 0esr](https://user-images.githubusercontent.com/46764043/146126592-45abb2b8-ec9d-4636-94fb-725fa7f04105.jpg) myMPD v8.2 development reports 
`23:28:28 ERROR    webserver Browser does not support gzip compression
`

- myMPD development Timer is not displayed properly on IOS web browsers. Please see image below for details
![IOS-Screenshot](https://user-images.githubusercontent.com/46764043/146127611-6315346b-30e5-4c8b-bb21-19ef1c591d3b.jpg)
myMPD development is not reporting any errors.

I apologize for not filling out a proper bug report.

**Comment by jcorporation** [comment]
Thanks for the feedback.

I opened 3 issues: #639, #640, #641

For the v8.0 and v8.1 releases much of the backend code was refactored to be more future proof. v9.0 (devel) refactors much of the frontend code. This refactoring is necessary to remove older bad design choices and to keep the codebase maintainable. 

There are many changes in the codebase in this releases and many changes produces more bugs than adding only small features. But time to time there must be such an cleanup with the associated hickups.

After sorting out the last bugs from devel, v9.0 will be released and this series should be pretty stable. There is no code or design refactoring planed. 

If you feel that the v7 branch is more stable go with it, but I need early adopters for new and devel version, because I can not test all features on all combinations of backend / frontend devices.

**Reply by jcorporation** [reply]
All issues are now addressed. It would be great if you can test. I tested it only quickly.

**Reply by vernonjvs** [reply]
Based on my testing, these bugs can be closed Thanks very much!. I will keep the development build on one of my Raspberry Pi Zero' W's for further testing.
